blocks/voice_recognition.js

```diff
@@ -0,0 +1,41 @@
+Blockly.Blocks['text_to_speech'] = {
+  init: function() {
+    this.appendDummyInput()
+        .appendField("Text to speech");
+    this.appendDummyInput()
+        .appendField("Command:")
+        .appendField(new Blockly.FieldTextInput("start"), "command");
+    this.setColour(210);
+    this.setTooltip('');
+    this.setPreviousStatement(true);
+    this.setNextStatement(true);
+    this.setHelpUrl('http://www.shadowrobot.com/');
+  }
+};
+
+Blockly.Blocks['word_list'] = {
+  init: function() {
+    this.appendDummyInput()
+        .appendField("Accepted words:");
+    this.appendDummyInput()
+        .appendField("Word 1:")
+        .appendField(new Blockly.FieldTextInput("start"), "word_1");
```

Comment:
should we have any defaults or just leave them blank?or we could fill them all with commands like pause ..